src/main/java/nl/novi/techiteasy1121/dtos/TelevisionDto.java

```diff
@@ -0,0 +1,181 @@
+package nl.novi.techiteasy1121.dtos;
+
+public class TelevisionDto {
```

Comment:
Deze DTO is prima, maar dit is alleen de output dto. Er ontbreekt nog een input dto waarin je ook validatie regels gebruikt (en waar je geen id hoeft te vragen)